engine/src/main/java/org/hibernate/validator/internal/util/ExecutablePropertyNodeNameProvider.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.internal.util;
+
+import java.io.Serializable;
+
+import org.hibernate.validator.spi.nodenameprovider.PropertyNodeNameProvider;
+
+public class ExecutablePropertyNodeNameProvider implements Serializable {
```

Comment:
We probably should change the name of this class so it matches more to its purpose. Let's first get settled with the main interface contract before any changes here.

Reply:
So it seems that we don't actually need this class.